.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,177 @@
+          MYSQL_DATABASE: fiber
+          MYSQL_USER: username
+          MYSQL_PASSWORD: password
+          MYSQL_ROOT_PASSWORD: password
+        ports:
+          - '3306:3306'
+        options: >-
+          --health-cmd "mysqladmin ping" --health-interval 10s --health-timeout
+          5s --health-retries 5
+      postgres:
+        image: 'postgres:latest'
+        ports:
+          - '5432:5432'
+        env:
+          POSTGRES_DB: fiber
+          POSTGRES_USER: username
+          POSTGRES_PASSWORD: "pass#w%rd"
+        options: >-
+          --health-cmd pg_isready --health-interval 10s --health-timeout 5s
+          --health-retries 5
+    steps:
+      - name: Fetch Repository
+        uses: actions/checkout@v4
+
+      - name: Install Go
+        uses: actions/setup-go@v4
+        with:
+          # NOTE: Keep this in sync with the version from go.mod
+          go-version: "1.20.x"
+
+      - name: Install Azurite
+        run: |
+          docker run -d -p 10000:10000 mcr.microsoft.com/azure-storage/azurite azurite-blob --blobHost 0.0.0.0 --blobPort 10000
+      - name: Install Coherence
+        run: |
+          docker run -d -p 1408:1408 -p 30000:30000 ghcr.io/oracle/coherence-ce:22.06.5
+          sleep 30
+      - name: Install couchbase
+        run: |
+          docker run --name couchbase -d -p 8091-8097:8091-8097 -p 9123:9123 -p 11207:11207 -p 11210:11210 -p 11280:11280 -p 18091-18097:18091-18097 couchbase:enterprise-7.1.1
+          sleep 10
+          docker exec --tty couchbase couchbase-cli cluster-init -c localhost:8091 --cluster-username admin --cluster-password 123456 --cluster-ramsize 256 --services data
+          sleep 10
+          docker exec --tty couchbase couchbase-cli bucket-create -c localhost:8091 --username admin --password 123456 --bucket fiber_storage --bucket-type couchbase --bucket-ramsize 100 --enable-flush 1
+      - name: Install etcd
+        run: |
+          docker run -d --name Etcd-server \
+            --publish 2379:2379 \
+            --publish 2380:2380 \
+            --env ALLOW_NONE_AUTHENTICATION=yes \
+            --env ETCD_ADVERTISE_CLIENT_URLS=http://etcd-server:2379 \
+            bitnami/etcd:latest
+      - name: Install MinIO
+        run: |
+          docker run -d --restart always -p 9000:9000 --name storage-minio -e MINIO_ROOT_USER='minio-user' -e MINIO_ROOT_PASSWORD='minio-password' minio/minio server /data
+      - name: Generate TLS certs
+        run: ./.github/scripts/gen-test-certs.sh
+
+      - name: Add Custom CA cert
+        run: sudo cp /home/runner/work/storage/storage/tls/ca.crt /usr/local/share/ca-certificates/custom.crt
+
+      - name: Trust Custom CA Cert
+        run: sudo update-ca-certificates
+
+      - name: Setup Redis
+        uses: shogo82148/actions-setup-redis@v1
+        with:
+          redis-version: '7.x'
+          auto-start: 'false'
+
+      - name: Run Redis
+        run: |
+          redis-server --tls-port 6380 --port 6379 \
```

Comment:
All the other DB's are not using TLS, we should remove it for the Benchmark of Redis and Ruedis.

TLS is known to cause performance issues for Redis in terms of Op/sec

Reply:
Seems the TestStore doesnt use TLS, so we dont need to do all this setup for the Benchmarks